### ISSUE TITLE
Accent Color <meta>

### ISSUE THREAD

**Issue by andreaboi83** [Swrve/swrve-android-sdk#124]
Hi. I'm trying to using new meta tag for setting up accent color in L+ notifications, but it doesn't work.

I think the problem is on these lines of code (83-87 in SwrveGcmNotification class):

```
int accentColor = metaData.getInt(SWRVE_PUSH_ACCENT_COLOR_METADATA, -1);
if (accentColor < 0) {
      ...
}
```

The value of accentColor from metadata is a resource identifier, so expected int color is the result of:

`ContextCompat.getColor(context, accentColor);`

In SwrveGcmHandler (line 147), you should do:

`mBuilder.setColor(ContextCompat.getColor(context, notificationHelper.accentColor));`

instead of:

`mBuilder.setColor(notificationHelper.accentColor);`

I'm sorry for my bad English...thanks in advance!

**Comment by dominicmarmionswrve** [comment]
Hi @andreaboi83, thanks for pointing this out. Yes indeed, I think you are correct. We'll target your fix for this in our next release! Thanks again! :beers:

**Comment by Sergio-Mira** [comment]
Thanks again for the headsup! We have a fix for this in the upcoming Android SDK 4.3, expected to be released by the end of the month:
https://github.com/Swrve/swrve-android-sdk/pull/137
